ui/window.go

```diff
@@ -287,9 +292,15 @@ func (win *Window) popPage() bool {
 	previousPage := win.pageBackStack[previousPageIndex]
 	win.pageBackStack = win.pageBackStack[:previousPageIndex]
 
+	// prevent page from resetting main page OnNavigatedTo data as restore page
+	// use the changePage() as using the popFragement() or changeFragement()
+	// renders the restore ui on a much smaller layout and distorts the display
```

Comment:
```
Avoid resetting data on the main page when returning to the
Main page from the Restore page. The Restore page was opened
using changePage() instead of changeFragment() because the
latter renders the restore page UI on a much smaller area and
distorts the display. Also, the Main page's data was not cleared
when the Restore page was opened via changePage().
```